### ISSUE TITLE
Does not allow to scroll any scrollable content inside Slide

### ISSUE THREAD

**Issue by theKashey** [alvarotrigo/react-fullpage#19]
Example - https://codesandbox.io/s/m5qjp6q0o9
Second slide contain scrollable content, but you cannot scroll it, as long any scroll will cause Slide change.

I think fullpage should or allow internal elements to be scrolled, and move to the next step only when scroll reaches the edge, or being able to disable any behavior by the user command.

**Comment by cmswalker** [comment]
@theKashey it seems to work with `overflow: auto` does that work for your use case?

**Comment by theKashey** [comment]
It will work when you will __change__ `scroll` to `auto` due to strange behavior, I've pointed in another ticket.
Try changing the code, and opening example in a separate tab - it still will do "full page scroll" on any scroll.

**Comment by cmswalker** [comment]
Not sure I follow, we have this in a separate issue https://github.com/alvarotrigo/react-fullpage/issues/18

Are you saying the HMR issue is related to the scroll issue?

**Comment by theKashey** [comment]
I think you changed overflow property, and example did work for you. So - I am saying that you actually break an example due to #18, and that's why scrollable container start working, but you could not leave that (odd) frame.

**Comment by cmswalker** [comment]
Ok, so perhaps we can close this issue and address https://github.com/alvarotrigo/react-fullpage/issues/18 ? Overflow inside of slides seems to be ok

**Comment by theKashey** [comment]
Not sure - I could not scroll content inside scrollable elements - it causes everything to scroll.
I am not sure how _this_ full page works, but for your version I have to implement something like this - https://github.com/frontendier/fullpage-react/blob/master/lib/components/Fullpage.js#L358 - ie completely disable listeners.
No other easy way :)
Complex way is something like https://github.com/theKashey/react-locky/blob/master/src/handleScroll.js - collect all scroll available, and scroll only after reaching the end, as I mentioned above.
Not sure how this possible with fullpage, but it does support "full page scroll"

**Comment by cmswalker** [comment]
Sorry I'm a bit confused now. Let's not reference the old project that is no longer maintained. 

I am able to get overflow scroll working in this project both in the codesandbox link and locally. 

Once the overflow scroll limit is exceeded, a fullpage scroll begins. Is this not the same behavior you are experiencing?

**Comment by cmswalker** [comment]
The more i futz with the codesandbox I think you're onto something that this + the HMR issue could be related. Going to leave both open for now and will look into this asap

**Comment by alvarotrigo** [comment]
> Not sure - I could not scroll content inside scrollable elements - it causes everything to scroll.

Use the fullpage option [`normalScrollElements`](https://github.com/alvarotrigo/fullPage.js#options). 

See your updated example here:
https://codesandbox.io/s/qk5lo3674q

**Comment by theKashey** [comment]
1. Setup `normalScrollElements`
2. Create such element after Fullpage initialization
3. Delete such element
4. Fullpage is not working. Not scrolling at all.

**Comment by alvarotrigo** [comment]
i see. So you are dynamically creating the element linked to "normalScrollElements" ?

Mmm seems to be [working fine on fullPage.js](https://jsfiddle.net/alvarotrigo/ea17skjr/6/), but I'll check it on react-fullpage.js!

**Comment by cmswalker** [comment]
It could be a race condition between when the functionality of `normalScrollElements` fires and when the render-prop is ready. We can start there

**Comment by cmswalker** [comment]
@theKashey I updated the demo w/ `v0.0.9` https://codesandbox.io/s/myl7l3rpn8

It seems to be working, can you verify?

**Comment by theKashey** [comment]
@cmswalker - working fine for me!

**Comment by alvarotrigo** [comment]
Awesome!! I'll close the issue then! ;)
